users/gurtatiLND/website/webpage.html

```diff
@@ -11,14 +12,45 @@
             <div class="header_main">Links to the sections</div>
         </header>
         <main>
-            <section class="home">Home</section>
-            <section id="about" class="about sec-pad">About</section>
-            <section id="projects" class="projects sec-pad">Projects</section>
-            <section id="contact" class="contact sec-pad">Contact</section>
+            <section class="home">
+                <div class="home_content">
+                    <h1 class="heading_home">HI, I'M TATIANA GUROVA</h1>
+                    <div class="home_info">
+                        <p class="text_home">Frontend Developer</p>
+                    </div>
+                    <div class="home_link">
+                        <a href="">Projects</a>
+                    </div>
+                </div>
+            </section>
+            <section id="about" class="about sec-pad">
+                <div class="main_container">
+                    <h2 class="heading about">ABOUT</h2>
+                    <div class="about_content">
+                        <p>How I became a frontend dev</p>
+                    </div>
+                </div>
+            </section>
+            <section id="projects" class="projects sec-pad">
+                <div class="main_container">
+                    <h2 class="heading projects">PROJECTS</h2>
+                    <div class="projects_content">
+                        <p>My projects:</p>
+                    </div>
+                </div>
+            </section>
+            <section id="contact" class="contact sec-pad">
+                <div class="main_container">
+                    <h2 class="heading contact">CONTACT</h2>
```

Comment:
kebab-case please

Reply:
This section has styles from the "heading" and "contact" classes.